service/src/integrationTest/resources/fixtures/caseDataWithMandatoryFields.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
why can't we use populate-case-details.json here?

Reply:
I would reuse whatever we have already, don't add new files. If there's some tech debt let's solve it separately

Reply:
I agree, we don't need new files if they are the same, the less the better. But at the same time don't make your PR bigger than it needs to be and you can clean up later!